From 4c7a9c51dad688061041ddca0e40ead76f241661 Mon Sep 17 00:00:00 2001
From: наб <nabijaczleweli@gmail.com>
Date: Thu, 18 Jun 2020 16:49:16 +0200
Subject: [PATCH] Only use thin LTO for release builds

Full LTO breaks [dh_]dwz on amd64:
  dwz: debian/systemd-zram/usr/lib/systemd/system-generators/zram-generator: Couldn't find DIE referenced by DW_AT_abstract_origin
  dh_dwz: error: dwz -- debian/systemd-zram/usr/lib/systemd/system-generators/zram-generator returned exit code 1

Upstream: https://github.com/rust-lang/rust/issues/66118
---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index c7cb750..16c1106 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -18,4 +18,4 @@ tempfile = "3"
 fs_extra = "1.1"

 [profile.release]
-lto = true
+lto = "thin"
--
2.27.0
